extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -476,6 +477,22 @@ export abstract class AzureAuth implements vscode.Disposable {
 
 	//#region network functions
 
+	private async makeGetRequest(url: string, token: string): Promise<AxiosResponse<any>> {
```

Comment:
Instead of having any here it would be a bit easier to have this be a generic function and have the response be of type T - that way the caller doesn't have to cast the response data themselves.